### DISCUSSION TITLE
browsershot configuration

### DISCUSSION THREAD

**Discussion by RVP04** [spatie/laravel-pdf#147] [Q&A]
I am getting this error in my production server
```
The command "PATH=$PATH:/usr/local/bin NODE_PATH=`npm root -g` node '/var/www/xxx/vendor/spatie/browsershot/src/../bin/browser.cjs' '
```

Eventhough i have installed the puppeteer package globally. I dont know what i am missing here.

Server: Almalinux 9
Laravel : 10.x
PHP: 8.2.19

**Comment by mohamed-salah-tawfik** [comment]
any updates here?

**Comment by mohamed-salah-tawfik** [comment]
I found a method named `withBrowsershot` at the `PdfBuilder` that you can use to set the binaries path

````
return pdf()
    ->withBrowsershot(function (Browsershot $shot) {
        $shot->setNodeBinary('YOUR-CUSTOM-PATH/node')
            ->setNpmBinary('YOUR-CUSTOM-PATH/npm');
    })
    ->view('html-view')
    ->name('file-name.pdf')
    ->download();
```

**Comment by RVP04** [comment]
i am already using browsershot and made the changes but sill facing same issue
